Tests/SonarQube.MSBuild.Tasks.IntegrationTests/SonarQube.MSBuild.Tasks.IntegrationTests.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SonarQube.MSBuild.Tasks.IntegrationTests</RootNamespace>
     <AssemblyName>SonarQube.Integration.Tasks.IntegrationTests</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
Miscellaneous: changing the framework version to the same as all of the other projects